logcat/src/nativeMain/kotlin/logcat/outerClassSimpleNameInternalOnlyDoNotUseKThxBye.kt

```diff
@@ -0,0 +1,15 @@
+package logcat
+
+private fun String?.stripCompanions(): String? {
+  return if (this.isNullOrEmpty() || this == "Companion") null else this
+}
+
+actual fun Any.outerClassSimpleNameInternalOnlyDoNotUseKThxBye(): String {
```

Comment:
Should do some visibility review. This gets exposed to the framework header, I'd assume unnecessarily

```objc
__attribute__((objc_subclassing_restricted))
__attribute__((swift_name("OuterClassSimpleNameInternalOnlyDoNotUseKThxByeKt")))
@interface LogcatOuterClassSimpleNameInternalOnlyDoNotUseKThxByeKt : LogcatBase
+ (NSString *)outerClassSimpleNameInternalOnlyDoNotUseKThxBye:(id)receiver __attribute__((swift_name("outerClassSimpleNameInternalOnlyDoNotUseKThxBye(_:)")));
@end;
```

Reply:
Updated that particular function to be `@PublishedApi internal actual`.  I don't see it showing up in my local build's framework header now, as far as I can tell (I'm not an iOS developer; primarily Android)